src/gordon_gcp/clients/gdns.py

```diff
@@ -61,6 +61,8 @@ class GCPResourceRecordSet:
 
     Args:
         name (str): Name/label.
+        kind (str): ID for what kind of DNS resource this is. For example
```

Comment:
Isn't the `kind` more of a "what kind of GCP resource" the resource is ? (As in, it's a GCP thing and not a Google DNS thing).

Reply:
ah yeah - i'll switch it to GCP. I put 'DNS' since it's only dealing with the Google DNS Resource record set API, but having GCP resource here makes sense too.